### DISCUSSION TITLE
Bin Datei für ESP-Flash-Tool nicht auffindbar

### DISCUSSION THREAD

**Discussion by DonCanDo** [dewenni/ESP_Buderus_KM271#112] [Q&A]
Hey,

ich versuche schon seid gestern die Firmware irgendwie auf das KM217 zu flashen. Allerdings finde ich die bin Datei nirgends.

**Comment by the78mole** [comment]
In den Releases (recht Spalte in der Repo-Hauptansicht):
https://github.com/dewenni/ESP_Buderus_KM271/releases

**Comment by DonCanDo** [comment]
Hey Daniel,

danke noch mal. Zum Fehlerausschluss muss ich dann noch mal ein neues Board kaufen.

Die Software läuft so gar nicht. Muss Dwenni leider mit Meldung voll ballern ich versuche hier schon alles mögliche dass irgendwas stabil läuft.


Von: Daniel Glaser ***@***.***>
Gesendet: Samstag, 12. Oktober 2024 19:13
An: dewenni/ESP_Buderus_KM271 ***@***.***>
Cc: Don ***@***.***>; Author ***@***.***>
Betreff: Re: [dewenni/ESP_Buderus_KM271] Bin Datei für ESP-Flash-Tool nicht auffindbar (Discussion #112)


In den Releases (recht Spalte in der Repo-Hauptansicht):
https://github.com/dewenni/ESP_Buderus_KM271/releases

-
